crits/core/handlers.py

```diff
@@ -657,6 +657,10 @@ def get_item_names(obj, active=None):
     :returns: :class:`crits.core.crits_mongoengine.CritsQuerySet`
     """
 
+    # Don't use this to get sources.
+    if isinstance(obj, SourceAccess):
+        return []
```

Comment:
It doesn't break anything, but that is by virtue of the fact that nothing calls ```get_item_names(SourceAccess)```. Any place that wants to enumerate sources uses the corresponding handler from that. This was added more as an anti-foot-shooting measure in case someone screws up and calls ```get_item_names(SourceAccess)```.